lib/src/Uncertainty/Distribution/KernelSmoothing.cxx

```diff
@@ -646,6 +719,10 @@ void KernelSmoothing::load(Advocate & adv)
   adv.loadAttribute("automaticLowerBound_", automaticLowerBound_);
   adv.loadAttribute("upperBound_", upperBound_);
   adv.loadAttribute("automaticUpperBound_", automaticUpperBound_);
+  if (adv.hasAttribute("useLogTransform_"))
+    adv.loadAttribute("useLogTransform_", useLogTransform_);
+  else
+    useLogTransform_ = false;
```

Comment:
the else can be removed as useLogTransform_ is already set to false in the default ctor